cmd/kubeadm/app/constants/constants.go

```diff
@@ -458,8 +458,8 @@ var (
 		18: "3.4.3-0",
 		19: "3.4.13-0",
 		20: "3.4.13-0",
-		21: "3.4.13-0",
-		22: "3.4.13-0",
+		21: "3.4.13-3",
```

Comment:
just something to note.

given we are not merging this PR in 1.21, this means that the 1.21 kubeadm binary will be with -0 and the 1.22 binary will be with -3.

this has caused problems in the past:
- where "kubeadm upgrade" did not work for revisions - e.g. from -0 to -3
can you please check if it works for you (we don't have kubeadm upgrade tests in CI here).
i'm pretty sure i fixed that, but we don't have an explicit e2e for that even in the kubeadm periodic tests.
- users have complained about it in the lines of "why does the etcd version differ between kubeadm releases for the same k8s version". the response is that this can happen due to security fixes, which we don't want to backport to the older k8s version.

Reply:
for `kubeadm upgrade`, let me test locally. 
Or we should add a upgrade test in CI later.

Reply:
this periodic test would catch the problem:
https://k8s-testgrid.appspot.com/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-upgrade-1-20-latest

we just don't have it in the k/k CI.